topics/metabolomics/tutorials/lcms-preprocessing/slides.html

```diff
@@ -151,20 +151,20 @@
 
 ## xcms - adjustRtime - Obiwrap algorithm
 
-![a scatter plot and some 3d plots are shown, their meaning is not clear.](../../images/lcmspreproc_slides_4.3.png)
+![A composed picture from the publication of the "obiwarp" algorithm.](../../images/lcmspreproc_slides_4.3.png)
 
 ---
 name:beforeafterrt
 
 ## xcms - adjustRtime - Output
 
-![In the first graphic two messy lines of points are shown with bandwidth=8, their times have been adjusted and now they line up perfectly into two individual peaks and are separated correctly.](../../images/lcmspreproc_slides_5.png)
+![A "before/after" picture related to the rt adjustment. In the first graph two messy lines of points are shown. It seems to be two ions, but because there are shifts in rt it is not possible to separate them, whatever the bw value. The second graph is after rt ajustment. Now the peaks points line up perfectly into two individual peaks and thus an appropriate bw value allows to separate the two peaks correctly.](../../images/lcmspreproc_slides_5.png)
```

Comment:
it would be nice if you could add the text of the four green boxes as well for the description